lib/tasks/migrator.js

```diff
@@ -64,5 +46,53 @@ module.exports = function runMigrations(context) {
         }
 
         return Promise.reject(error);
-    });
-}
+    }
+};
+
+module.exports.migrate = function runMigrations(context) {
+    const {dir} = context.instance;
+
+    const contentDir = path.join(dir, 'content');
+    const currentDir = path.join(dir, 'current');
+    let knexMigratorPromise;
+
+    const args = ['--init', '--mgpath', currentDir];
+
+    // If we're using sqlite and the ghost user owns the content folder, then
+    // we should run sudo, otherwise run normally
+    if (ghostUser.shouldUseGhostUser(contentDir)) {
+        const knexMigratorPath = path.resolve(dir, 'current/node_modules/.bin/knex-migrator-migrate');
+        knexMigratorPromise = context.ui.sudo(`${knexMigratorPath} ${args.join(' ')}`, {sudoArgs: '-E -u ghost'});
+    } else {
+        knexMigratorPromise = execa('knex-migrator-migrate', args, {
+            preferLocal: true,
+            localDir: path.join(dir, 'current')
+        });
+    }
+
+    return knexMigratorPromise.catch(errorHandler(context));
+};
+
+module.exports.rollback = function runRollback(context) {
+    const {dir} = context.instance;
+
+    const contentDir = path.join(dir, 'content');
+    const currentDir = path.join(dir, 'current');
+    let knexMigratorPromise;
+
+    const args = ['--force', '--version', context.instance.cliConfig.get('previous-version'), '--mgpath', currentDir];
```

Comment:
The `--version` option requires the new version of knex-migrator in Ghost.
We have to add a bridge here (execute only for v2). A little annoying, but not sure how else to solve.
This is the disadvantage of not having knex-migrator as dependency in the CLI.